These patches allow the man pages to be built using older version
of Sphinx. These patches may be dropped for more recent versions.
These components are not used for the man pages.

* sphnix.ext.napoleon appears in Sphinx 1.3
* sphinx.ext.autosectionlabel appears in Sphinx 1.4

* nbsphinx is only available via pip

* pathlib appears in Python 3

diff --git a/docs/conf.py b/docs/conf.py
index 506b948..3abac8c 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -17,8 +17,6 @@ import os
 
 import re
 
-from pathlib import Path
-
 ON_RTD = os.environ.get('READTHEDOCS') == 'True'
 if ON_RTD:
     print("ON RTD, THEREFORE INSTALLING HTCONDOR PACKAGE")
@@ -51,11 +49,8 @@ sys.path.append(os.path.abspath('extensions'))
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autosectionlabel',
     'sphinx.ext.intersphinx',
     'sphinx.ext.autodoc',
-    'sphinx.ext.napoleon',
-    'nbsphinx',
     'ticket',
     'macro',
     'macro-def',
